db/migrate/20170908172207_add_status_to_sample.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToSample < ActiveRecord::Migration[5.1]
+  def change
+    add_column :samples, :status, :string
```

Comment:
I couldn't figure out from this review -- what is the "status" of a sample used to capture?

Reply:
lgtm 
a few comments and questions.